homeassistant/components/abode/sensor.py

```diff
@@ -41,23 +41,16 @@ def __init__(self, data, device, sensor_type):
         """Initialize a sensor for an Abode device."""
         super().__init__(data, device)
         self._sensor_type = sensor_type
-        self._name = f"{self._device.name} {SENSOR_TYPES[self._sensor_type][0]}"
         self._device_class = SENSOR_TYPES[self._sensor_type][1]
-
-    @property
-    def name(self):
-        """Return the name of the sensor."""
-        return self._name
-
-    @property
-    def device_class(self):
-        """Return the device class."""
-        return self._device_class
-
-    @property
-    def unique_id(self):
-        """Return a unique ID to use for this device."""
-        return f"{self._device.device_uuid}-{self._sensor_type}"
+        self._attr_name = f"{device.name} {SENSOR_TYPES[sensor_type][0]}"
+        self._attr_device_class = device_class
```

Comment:
Sorry that is my bad, self._device_class I think can be removed and directly assigned here.